### ISSUE TITLE
Content Scripts get erased

### ISSUE THREAD

**Issue by tsarma** [browserstate/ajaxify#19]
I'm updating my main content (div#container) with ajaxify. This div may contain some script tag dynamically generated from CMS. But ajaxify is removing all scripts tags within the content by "$scripts.detach();". How about options to overcome it.
EDITED:
I was wrong, ajaxify reinsert the script back. was console.loggin at wrong place.
